src/menu-settings.c

```diff
@@ -59,6 +72,17 @@ static void brisk_menu_window_settings_changed(GSettings *settings, const gchar
         }
 }
 
+/* callback for changing applet settings */
+static void brisk_menu_applet_settings_changed(GSettings *settings, const gchar *key, gpointer v)
+{
+        BriskMenuApplet *self = v;
+        gchar *shortcut = g_settings_get_string(settings, key);
```

Comment:
This will lead to a segfault when changing the non-string keys (i.e boolean dark, etc.)
You've chained a watch on all key types so you cannot get the explicitly typed value without know what the type will be first.